.github/workflows/build.yaml

```diff
@@ -57,6 +64,8 @@ jobs:
     - name: Install Rust Toolchain Components
       run: |
         rustup component add clippy rustfmt
+        rustup target add ${{ matrix.target }}
+        rustup default `cat rust-toolchain`-${{ matrix.target }}
```

Comment:
I personally like `…` a little better, since it's the first form I learned. (Although it can't nest, which is a downside.) In this case I like that the backticks are visually distinct from `${{…}}`.